### ISSUE TITLE
The plugin doesn't work with the gradle shadow plugin

### ISSUE THREAD

**Issue by swamyrajamohan** [ryandens/javaagent-gradle-plugin#100]
When used with the gradle-shadow plugin and creating the shadowDistTar or shadowDistZip instead of the regular distZip and distTar provided by the application plugin the resultant distributions don't contain the **agent-libs** directory in the distribution (zip/tar).

tar produced by the distTar after applying the javaagent-gradle-plugin
**agent-libs**  # containing the javaagent lib
**bin**             # containing the startup scripts
**lib**              # containing the dependent libraries and application jar

tar produced by the distTar after applying the javaagent-gradle-plugin
#Misses the agent-libs folder containing javaagent lib
**bin**             # containing the startup scripts
**lib**              # containing the dependent libraries and application jar

```groovy
plugins {
    id 'java'
    id 'groovy'
    id 'application'
    id 'com.ryandens.javaagent-application' version '0.4.2'  
    id 'maven-publish'
    id 'com.github.jk1.dependency-license-report' version '2.5'
    id 'com.github.johnrengelman.shadow' version '8.1.1'
    id 'org.sonarqube' version '4.4.1.3373'
    id 'org.owasp.dependencycheck' version '9.0.9'
}

**Comment by swamyrajamohan** [comment]
Tried it with the newer version 0.5.1 as well and still the same issue

**Comment by ryandens** [comment]
👋 i wouldn't expect it to work with the artifacts built by the shadow task - feel free to make a PR!

**Comment by tajobe** [comment]
I think this is the same as #94, where I've included a potential workaround @swamyrajamohan

**Comment by ryandens** [comment]
closed as duplicate of #94
